ios/ReactTestApp/UIViewController+ReactTestApp.h

```diff
@@ -1,5 +1,5 @@
 #include <TargetConditionals.h>
-#if TARGET_OS_IOS
+#if TARGET_OS_IPHONE
```

Comment:
Can we use `__has_include` here?
```suggestion
#if __has_include(<UIKit/UIViewController.h>)
```